packages/lodestar-beacon-state-transition/src/block/operations/attesterSlashing.ts

```diff
@@ -45,5 +45,5 @@ export function processAttesterSlashing(
     }
   });
 
-  assert(slashedAny);
+  assert.true(slashedAny, "should slash any");
```

Comment:
What would be an appropriate assertion message here? I don't believe this one is clear to end users

Reply:
maybe "no slashable validators for attester slashing found"